src/main/java/yeelp/distinctdamagedescriptions/capability/impl/ShieldDistribution.java

```diff
@@ -36,7 +36,7 @@ public ShieldDistribution(Tuple<DDDDamageType, Float>... mappings) {
 	public DamageMap block(DamageMap fullDamage) {
 		this.getCategories().forEach((t) -> fullDamage.computeIfPresent(t, (k, v) -> {
 			float f = blockDamage(v, this.getWeight(k));
-			return f <= 0 ? null : f;
+			return f < 0 ? null : f;
```

Comment:
If the shield fully blocks the damage (i.e. `f <= 0`), I believe we actually do want that key removed from the map.  I think this is for a few reasons regarding adaptive resistance and other functionality, which expects that every key present in the map after the shield block is damage actually being inflicted.